ci/prow/config_knative.yaml

```diff
@@ -357,6 +363,14 @@ periodics:
     - auto-release: true
       dot-dev: true
       go113: true
+    - custom-job: integration-tests-on-latest-serving
+      command:
+      - "./test/presubmit-tests.sh"
```

Comment:
This is not a presubmit job, don't use presubmit-tests.sh.

Reply:
Sorry got the two messed, fixed.